kt/utils/godot-build-props/build.gradle.kts

```diff
@@ -43,3 +43,5 @@ publishing {
 }
 
 project.extra["artifacts"] = arrayOf("buildProps")
+
+tasks.withType<Copy> { duplicatesStrategy = DuplicatesStrategy.INCLUDE}
```

Comment:
What is this for ?

Reply:
Damn. Should have commented why i added that xD
Builds failed because of an issue with gradle 7.x
Something about property files from different modules IIRC.
I'll see if A it is still needed, and B if i find the issue again where i found this solution and comment it accordingly.

Reply:
Ah the problem was with how gradles copy task functions. If you copy multiple files at once you now need to explicitly set a duplication strategy: https://docs.gradle.org/7.0.2/dsl/org.gradle.api.tasks.Copy.html#org.gradle.api.tasks.Copy:duplicatesStrategy
And this is the case for the copy task in build props it seems. So i specified the duplication strategy explicitly here